db/seeds.rb

```diff
@@ -51,7 +51,7 @@
   },
   {
     application: "LTI Starter App",
-    tenant: "starter-app",
+    tenant: Rails.application.secrets.default_lti_key,
```

Comment:
I'd rather the secrets than env. If we want env, we should load the env in the secrets and still use the secrets here.